gradle/libs.versions.toml

```diff
@@ -22,7 +22,7 @@ mockito = "5.11.0"
 mockitoKotlin = "5.2.1"
 mockk = "1.13.10"
 node-gradle = "7.0.1"
-telemetryGenerator = "1.0.207"
```

Comment:
note: this should be updated to even a newer version.

This PR is pending to be merged https://github.com/aws/aws-toolkit-common/pull/737